### PR TITLE
dev-libs/libuev: add 2.4.1

### PR DESCRIPTION
**Pull request by jubalh**

### CODE REVIEW

File: dev-libs/libuev/libuev-2.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2022 Gentoo Authors
```
Comment (ConiKost) [inline dev-libs/libuev/libuev-2.4.1.ebuild:1]: Please update year.

File: dev-libs/libuev/libuev-2.4.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (ConiKost) [inline dev-libs/libuev/libuev-2.4.1.ebuild:4]: Please use EAPI=8

File: dev-libs/libuev/libuev-2.4.1.ebuild (hunk 1 above)
Comment (jubalh) [reply dev-libs/libuev/libuev-2.4.1.ebuild]: done

File: dev-libs/libuev/libuev-2.4.1.ebuild (hunk 2 above)
Comment (jubalh) [reply dev-libs/libuev/libuev-2.4.1.ebuild]: done

**Review by ConiKost — APPROVED** [review]

**Pull request closed without merging**